Odata-docs/odatalib/v7/edm/iedmtargetpath-for-annotations-target.md

```diff
@@ -0,0 +1,463 @@
+
+## Background
+
+Below is a simple Edm model with an out-of-line annotation that targets a model element.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+    <edmx:DataServices>
+        <!-- Model elements -->
+        <Schema Namespace="Default" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+            <EntityContainer Name="Container">
+                <EntitySet Name="Employees" EntityType="Ns.Employee" />
+            </EntityContainer>
+            <Annotations Target="Ns.Employee">
+                <Annotation Term="Ns.MyTerm" String="Name OutOfLine MyTerm Value" />
+            </Annotations>
+        </Schema>
+    </edmx:DataServices>
+</edmx:Edmx>
+```
+
+`IEdmTargetPath` allows us to define out-of-line annotations that target a path to a model element.
+
+Below are examples of paths to a model element.
+```
+MySchema.MyEntityContainer/MyEntitySet/MyProperty
+MySchema.MyEntityContainer/MyEntitySet/MyNavigationProperty
+MySchema.MyEntityContainer/MyEntitySet/MySchema.MyEntityType/MyProperty
+MySchema.MyEntityContainer/MyEntitySet/MySchema.MyEntityType/MyNavProperty
+MySchema.MyEntityContainer/MyEntitySet/MyComplexProperty/MyProperty
+MySchema.MyEntityContainer/MyEntitySet/MyComplexProperty/MyNavigationProperty
+MySchema.MyEntityContainer/MySingleton/MyComplexProperty/MyNavigationProperty
+```
+
+Below is a simple Edm model with an out-of-line annotation that targets a path to a model element.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+    <edmx:DataServices>
+        <!-- Model elements -->
+        <Schema Namespace="Default" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+            <EntityContainer Name="Container">
+                <EntitySet Name="Employees" EntityType="Ns.Employee" />
+            </EntityContainer>
+            <Annotations Target="Default.Container/Employees/Ns.Manager/EmployeeId">
+                <Annotation Term="Ns.MyTerm" String="Name OutOfLine MyTerm Value" />
+            </Annotations>
+        </Schema>
+    </edmx:DataServices>
+</edmx:Edmx>
+```
+
+## Create a .NET Core console application
+Create a .NET Core console application and import the `Microsoft.OData.Edm` nuget package:
+
+---
+
+# [Visual Studio](#tab/visual-studio)
+
+In the Visual Studio **Package Manager Console**:
+```powershell
+Install-Package Microsoft.OData.Edm
+```
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+```dotnetcli
+dotnet add package Microsoft.OData.Edm
+```
+
+---
+
+## Configure the Edm model
+Add a _SampleModel.cs_ and replace the code with the following code
+
+```csharp
+using Microsoft.OData.Edm;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AnnotationsTarget
+{
+    internal class SampleModel
+    {
+        public SampleModel()
```

Comment:
In my opinion, the only element that you need to add in your model for purposes of what you're trying to illustrate is `Customer` and the `Customers` entity set. You don't need everything else. It just makes your sample unnecessarily long, distracting and full of noise.